src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,36 @@
+package nextstep.qna.domain;
+
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Answers {
+
+    private List<Answer> answers = new ArrayList<>();
+
+    public void add(Answer answer) {
+        this.answers.add(answer);
+    }
+
+    public List<Answer> getAnswers() {
+        return answers;
+    }
+
+    public void validateAnswerOwner(NsUser loginUser) throws CannotDeleteException {
```

Comment:
```suggestion
    public void delete(NsUser loginUser) throws CannotDeleteException {
```
삭제 가능 여부를 확인하는 것이 핵심이 아니라 삭제가 핵심이지 않을까?